ConsumerApi/appsettings.json

```
@@ -14,16 +14,30 @@
                 "MinimumBackoff": 2,
                 "MaximumBackoff": 120
             }
+        },
+        "SqlDatabaseConfiguration": {
```

There is no `SqlDatabaseConfiguration` property

```suggestion
        "SqlDatabase": {
```